tmpl/wiki/weekly/2015-01-14.md

```diff
@@ -0,0 +1,149 @@
+14th Jan 2014: Project name, 4.02.1 support, TLS on Xen and Error handling
+
+### Agenda ###
+
+- "Mirage OS" or "MirageOS"?
+- Pull OCaml runtime/bindings into packages / OCaml 4.02.1 support
+- TLS on Xen
+- Error handling in Mirage
+- Planning for 3.0
+
+
+Attendees: Amir Chaudhry, Thomas Gazagnaire, David Kaloper, Thomas Leonard,
+Jon Ludlam, Anil Madhavapeddy, Hannes Mehnert, Richard Mortier, Dave Scott
+and Magnus Skjegstad
+
+
+### Notes ###
+
+#### "Mirage OS" or "MirageOS"? ####
+
+We need to settle on how we refer to the project and then be consistent in its
+use. In the early days it was referred to variously as 'Mirage', 'Open
+Mirage', 'Mirage OS' and 'MirageOS'.  When we became a
+[Xen Incubated project][incubation], we had a brief discussion about the
+'official' name (for press releases etc) and we settled on 'Mirage OS'. 
+However, we haven't always been consistent with the way we use the term.  So
+far all the official materials such as new releases, have referred to
+"Mirage OS" (with a space), whereas on Twitter and our slides we see 'MirageOS'
+(without a space) being used. We should stick to only one of these.
+
+**Why does this matter?** - As we attract more users and raise more awareness,
+being consistent in how we represent ourselves will begin to matter more. The
+name is just one of these aspects and even though it may seem trivial, it *is*
+the name of the project we're discussing here. At this stage, there should be
+no confusion about how it's written.
+
+After a brief discussion, it's clear that most of the team are less concerned
+about the exact choice rather than being consistent. Only Amir and Anil had
+opinions to share and they both proposed that we use 'MirageOS' from now on.
+The reasons are that users have independently used this form in tweets and
+posts, it is easier to search for, and it's better to go with the flow as we
+still can.
+
+Overall, we unanimously decided on two things: (1) to move forward with
+'**MirageOS**' as the official name of the project and (2) be much more
+consistent in using it on our written material.
+
+In terms of practical steps, Amir will update things as he sees them on the
+website and written materials. Anil suggested that we may start using a
+different domain for the site too ([mirage.io](http://mirage.io)).
+
+[incubation]: http://lists.xenproject.org/archives/html/mirageos-devel/2013-01/msg00081.html
+
+#### OCaml 4.02.1 support ####
+
+Anil's working on this ([mirage/mirage-platform#115][115]) and thanks ThomasL
+for his patches. Hasn't had time to merge yet but will do soon. Might not
+rebase it and hopes that won't be too much of a problem for anyone.  In
+general we shouldn't do that but this one would take a while to rebase and
+would prefer to merge quickly.  The functor inline patch from 4.02.1 will have
+big performance boost but we should clearly state that we're not dropping
```

Comment:
the inline patch is not in 4.02.1 yet but it applies cleanly on trunk. The work done to port Mirage to 4.02.1 makes the port to trunk and to the inline branch of trunk trivial (hopefully).